tests/test_field.py

```diff
@@ -188,33 +188,53 @@ def test_field_svd():
 		assert np.allclose(s[..., i], svd2[1])
 		assert np.allclose(vh[..., i], svd2[2])
 
-def test_grid_hashing():
+def test_grid_hashing_and_comparison():
 	grid1 = make_pupil_grid(128)
 
 	grid2 = CartesianGrid(SeparatedCoords(copy.deepcopy(grid1.separated_coords)))
 	assert hash(grid1) != hash(grid2)
+	assert grid1 != grid2
+	assert grid2 != grid1
 
 	grid3 = CartesianGrid(UnstructuredCoords(copy.deepcopy(grid1.coords)))
 	assert hash(grid1) != hash(grid3)
+	assert grid1 != grid3
+	assert grid3 != grid1
 
```

Comment:
Add an assert grid2 != grid3. Otherwise we will miss a code check for unstructure vs separated coords.

Reply:
Done. Also added the reverse (which goes through a different code path.